cds_dojson/marc21/fields/books/multipart.py

```diff
@@ -143,11 +250,16 @@ def migration(self, key, value):
                 raise_exception=True,
                 subfield='n'
             )
-            volume_obj = {'title': val_p,
-                          'volume': volume_number,
-                          }
-            _migration['volumes'].append(volume_obj)
-    if not _series_title:
+            obj = {'title': val_p or volume_title}
+            if val_y:
+                if re.match('\\d+', val_y) and 1800 <= int(val_y) <= 2021:
```

Comment:
why the years constraint, are you sure we don't have older books? :)

Reply:
Well you're right, it may be too tight. I wanted to be sure to avoid false positives.